### ISSUE TITLE
Provide Productive Preference for Activating Multi-Monitor HiDPI Support

### ISSUE THREAD

**Issue by HeikoKlare** [vi-eclipse/Eclipse-Platform#92]
As a user of the Eclipse IDE (or any other Eclipse-RCP-based product) on Windows, I want to be able to (de-)activate multi-monitor HiDPI support within the tool instead of defining a startup flag.
To this end, we should add an Eclipse preference for enabling the multi-monitor HiDPI support on Windows. This allows to easily test and adopt the new functionality. It also gives us the chance to quite easily enable this preference by default at some point in time, when the new support is mature enough to be used by default (or to enable it for I-/M-builds so that a larger user base tests the new functionality).

Preference description could be something like `Enable HiDPI support for multiple monitors (Windows only)" or the like.

This should also remove the existing experimental preference.

**Comment by amartya4256** [comment]
This is also finished. @HeikoKlare @fedejeanne @akoch-yatta

**Comment by akoch-yatta** [comment]
No, this is not the experimental preference. That would be the final one, so it is active by default for everyone

**Comment by HeikoKlare** [comment]
Additional task: write information via cross-projects (or now ide) mailing list.
